.github/workflows/ci.yml

```diff
@@ -454,9 +454,6 @@ jobs:
       matrix:
         include:
           # These must match the versions of postgres used in the docker-compose.yml
-          - img_ver: 11-3.0-alpine
-            args: postgres
-            sslmode: disable
```

Comment:
I might be missing something, but I don't think we need to drop this.
https://github.com/maplibre/martin/pull/1774 only had problems with gdal as far as I can see. 🤔

Reply:
yeah, i think v11 and even v12 might be ok to be dropped